### ISSUE TITLE
Bulk Sequence Sheet Entry

### ISSUE THREAD

**Issue by vtrlx** [bcndp/dandelion-features#3]
During E-Day, entering Sequence numbers into Dandelion was a slow, gruesome process.

Our riding's sequence sheets were filled in by hand at the polls, and the numbers weren't ordered in any way, so when entering the sheets into Dandelion, our volunteers would have to scroll up and down the page many times just to complete a single sheet. Entering a single poll (consisting of half a dozen sheets) would take us up to 15 minutes.

Suggested fix: implement a per-poll bulk-entry interface that allows users to type out a series of sequence numbers (separated by spaces or commas) to be marked as 'Voted', as an alternative to the existing "hunt-and-peck" style of data entry.

**Comment by craiig** [comment]
Thanks Victoria! Great feedback. This is definitely on the list to improve. I expected us all to get bubblesheets but seems like we all got lists. :( I’ll make sure there’s a few different ways to enter them in the future!

**Comment by vtrlx** [comment]
Is Dandelion accepting contributions/contributors? Because I am interested in helping out.

**Comment by craiig** [comment]
Yes! It’s just been me so far. I’m still working out the best way to work with other folks on this but I’d love the help. Can I send you a message when I’ve got that part figured out? Maybe in a month or so.

**Comment by vtrlx** [comment]
May I e-mail you tonight?

**Comment by craiig** [comment]
Sure, craiig@gmail.com
